src/algebra/periodic.lean

```diff
@@ -66,6 +68,23 @@ lemma periodic.div [has_add α] [has_div β]
   periodic (f / g) c :=
 by simp * at *
 
+@[to_additive]
+lemma periodic.prod [has_add α] [comm_monoid β] {ι : Type*} {f : ι → α → β} {s : finset ι}
```

Comment:
Can you add the list.prod and multiset.prod versions too? Usually everything follows trivially from the list result.

Reply:
Now done!